packages/mainsail/source/fee.service.ts

```diff
@@ -18,22 +25,21 @@ export class FeeService extends Services.AbstractFeeService {
 
 	public override async all(): Promise<Services.TransactionFees> {
 		const node = await this.#request.get("node/fees");
-		const type = await this.#request.get("transactions/fees");
+		const dynamicFees: Fees = node.data.evmCall;
 
-		const staticFees: object = type.data;
-		const dynamicFees: object = node.data;
+		const fees = this.#transform(dynamicFees);
 
 		return {
-			delegateRegistration: this.#transform("validatorRegistration", 1, staticFees, dynamicFees),
-			delegateResignation: this.#transform("validatorResignation", 1, staticFees, dynamicFees),
-			ipfs: this.#transform("ipfs", 1, staticFees, dynamicFees),
-			multiPayment: this.#transform("multiPayment", 1, staticFees, dynamicFees),
-			multiSignature: this.#transform("multiSignature", 1, staticFees, dynamicFees),
-			secondSignature: this.#transform("secondSignature", 1, staticFees, dynamicFees),
-			transfer: this.#transform("transfer", 1, staticFees, dynamicFees),
-			usernameRegistration: this.#transform("usernameRegistration", 1, staticFees, dynamicFees),
-			usernameResignation: this.#transform("usernameResignation", 1, staticFees, dynamicFees),
-			vote: this.#transform("vote", 1, staticFees, dynamicFees),
+			delegateRegistration: fees,
```

Comment:
I had no choice but return fee options for specific types the method signature would n't let me do otherwise.

```
export interface TransactionFees {
	// Core
	transfer: TransactionFee;
	secondSignature: TransactionFee;
	delegateRegistration: TransactionFee;
	usernameRegistration: TransactionFee;
	usernameResignation: TransactionFee;
	vote: TransactionFee;
	multiSignature: TransactionFee;
	ipfs: TransactionFee;
	multiPayment: TransactionFee;
	delegateResignation: TransactionFee;
}
```